### PR TITLE
HDDS-3798. Display more accurate timestamp in recon Web

### PR DESCRIPTION
**Pull request by runitao**
## What changes were proposed in this pull request?

Display more accurate timestamp in recon Web. Currently the web only show timestamp in minutes.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-3960

## How was this patch tested?

Test with `pnpm run dev`

Here is the result with this patch.
image

### CODE REVIEW

File: hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/datanodes/datanodes.tsx [hunk 1]
```diff
@@ -135,7 +135,7 @@ const COLUMNS = [
     isVisible: true,
     sorter: (a: IDatanode, b: IDatanode) => a.lastHeartbeat - b.lastHeartbeat,
     render: (heartbeat: number) => {
-      return heartbeat > 0 ? moment(heartbeat).format('lll') : 'NA';
+      return heartbeat > 0 ? moment(heartbeat).format('YYYY/MM/DD HH:mm:ss') : 'NA';
```
Comment (vivekratnavel) [inline hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/datanodes/datanodes.tsx:138]: Can we use `ll LTS`?
```suggestion
      return heartbeat > 0 ? moment(heartbeat).format('ll LTS') : 'NA';
```

File: hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/datanodes/datanodes.tsx [hunk 2]
```diff
@@ -197,7 +197,7 @@ const COLUMNS = [
     isVisible: false,
     sorter: (a: IDatanode, b: IDatanode) => a.setupTime - b.setupTime,
     render: (uptime: number) => {
-      return uptime > 0 ? moment(uptime).format('lll') : 'NA';
+      return uptime > 0 ? moment(uptime).format('YYYY/MM/DD HH:mm:ss') : 'NA';
```
Comment (vivekratnavel) [inline hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/datanodes/datanodes.tsx:200]:
```suggestion
      return uptime > 0 ? moment(uptime).format('ll LTS') : 'NA';
```

File: hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/pipelines/pipelines.tsx [hunk 3]
```diff
@@ -114,7 +114,7 @@ const COLUMNS = [
     dataIndex: 'lastLeaderElection',
     key: 'lastLeaderElection',
     render: (lastLeaderElection: number) => lastLeaderElection > 0 ?
-      moment(lastLeaderElection).format('lll') : 'NA',
+      moment(lastLeaderElection).format('YYYY/MM/DD HH:mm:ss') : 'NA',
```
Comment (vivekratnavel) [inline hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/pipelines/pipelines.tsx:117]:
```suggestion
      moment(lastLeaderElection).format('ll LTS') : 'NA',
```

File: hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/autoReloadPanel/autoReloadPanel.tsx [hunk 4]
```diff
@@ -44,7 +44,7 @@ class AutoReloadPanel extends React.Component<IAutoReloadPanelProps> {
         <Tooltip
           placement='bottom' title={moment(lastUpdated).format('lll')}
```
Comment (vivekratnavel) [inline hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/autoReloadPanel/autoReloadPanel.tsx:45]: Can we also update the tooltip to include seconds?
```suggestion
          placement='bottom' title={moment(lastUpdated).format('ll LTS')}
```

**Comment by runitao** [comment]
Thanks @vivekratnavel for review. I have changed the timestamp format.

image

**Comment by adoroszlai** [comment]
Thanks @runitao for the improvement and @vivekratnavel for the review.

**Pull request merged**